tensorflow_addons/layers/tests/embedding_bag_test.py

```diff
@@ -60,7 +60,9 @@ def test_forward(input_shape, input_dim, dtype, indices_dtype, combiner):
         indices,
         weights,
     )
-    test_utils.assert_allclose_according_to_type(expected, output)
+    test_utils.assert_allclose_according_to_type(
+        expected, output, half_rtol=1e-2, half_atol=1e-2
+    )
 
 
 @pytest.mark.with_device(["cpu", "gpu"])
```

Comment:
Use @pytest.mark.usefixtures("maybe_run_functions_eagerly") instead of graph mode

Reply:
Done!